lib/src/CmsBootstrap.php

```diff
@@ -171,10 +171,6 @@ public function bootCms() {
     call_user_func([$this, $func],
       $cms['path'], $this->options['user'], $this->options['httpHost']);
 
-    if (PHP_SAPI === "cli") {
-      error_reporting(1);
-    }
-
```

Comment:
This is more likely to be doing something. Different CMS's are likely to switch error-reporting in different ways. (*And if they have primary responsibility for processing the request, then power to them.*) But from cv-cli pov, we just want access to their APIs -- and then the CLI UX should be consistent wrt to error-handling.

Looking at this again, I'm a little on the fence. Perhaps this value should be tuned-in more to the verbosity level...

Reply:
On a live site, it's probably set to 1 to begin with. On a dev site, it's probably E_ALL. I'm not clear on why it can't just be left at whatever the surrounding environment has. But yes it could work to set error_reporting(1) here, but if you specify enough 'v's then skip setting it (and/or maybe force E_ALL with 3 v's?)